netbox/netbox/views/generic.py

```diff
@@ -665,7 +665,10 @@ class ImportForm(BootstrapMixin, Form):
                 from_form=self.model_form,
                 widget=Textarea(attrs=self.widget_attrs)
             )
-
+            upload_csv = CSVFileField(
```

Comment:
This field needs a human-friendly label (e.g. "CSV file")

Reply:
Also, `csv_file` might be a better name than `upload_csv`.

Reply:
Renamed.